patch.diff

```diff
@@ -0,0 +1,44 @@
+diff --git a/PhpcsChanged/GitWorkflow.php b/PhpcsChanged/GitWorkflow.php
```

Comment:
I'm guessing this file was not intentional.

Reply:
Gah, `git add .` 🤦‍♂ Lemme rebase and update the PR

Reply:
Removed, sorry for the hassle and thanks for catching that!